src/tid-ext.adoc

```diff
@@ -0,0 +1,104 @@
+== "Zstid Extension for Thread Identification
+
+{tid_ext_name} is an optional extension to the RISC-V base ISA.
+Implementations that support {cheri_base_ext_name} and {tid_ext_name}.
+define a variant of the CHERI ISA that allows for software compartmentalization
+of CHERI programs.
```

Comment:
I'd argue it's possible without as well, so maybe quantify that with "improved"/"more efficient"?